Shoko.Server/Server/ShokoServer.cs

```diff
@@ -658,7 +658,11 @@ void WorkerFileEvents_DoWork(object sender, DoWorkEventArgs e)
 
                                 foreach (string file in files)
                                 {
-                                    if (FileHashHelper.IsVideo(file))
+                                    if (ServerSettings.Instance.Import.Exclude.Any(s => file.Contains(s)))
+                                    {
+                                        logger.Info("Import exclusion, skipping file {0}", file);
```

Comment:
This won't break out of the loop, you might need to add `continue;` after the log line.

Reply:
I left that out because I assume it still needed to flow down to>>
`queueFileEvents.Remove(evt);`

It looks like an event loop/thread from what I could tell

Reply:
After spending some time looking over WorkerFileEvents_DoWork() and gaining a better understanding of it. It makes sense for it to just flow down/on.